src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.io.File;
+
+/**
+ * Duke.java
+ * CS2103T
+ * @author Gabriel Ong
+ *
+ * This program is an interactive task list that takes in several
+ * preset commands from the user to create tasks, view the list of
+ * tasks and mark each of it as completed. This class contains the
+ * main method and is responsible for all input/output and Task creation.
+ *
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Returns a instance of duke.
+     * @param filePath a filepath indicating the storage directory for duke
```

Comment:
I think there would need to be a ending period for each line of JavaDocs comments, as there might be checkstyle error.

Reply:
Parameters actually don't need periods at the end, the official javadocs also do not actually add periods for their parameters. Descriptions do however!
<img width="586" alt="Screenshot 2019-09-04 at 10 13 49 AM" src="https://user-images.githubusercontent.com/41467676/64220655-b37e7380-cefc-11e9-9c18-fb89900ce5bc.png">

Reply:
Oh wow that's pretty specific thanks for sharing!